src/index.tsx

```diff
@@ -35,12 +35,22 @@ function concatTranscripts(...parts: string[]) {
     .trim();
 }
 
-export function useSpeechRecognition(options: SpeechRecognitionOptions = defaultOptions): SpeechRecognitionUtils {
+export function useSpeechRecognition(
+  userOptions: Partial<SpeechRecognitionOptions> = defaultOptions,
```

Comment:
I think we can make the default just an empty `{}`. Since we are combining them below it no longer makes sense to default to `defaultOptions` here.